src/OpenColorIO/GpuShaderUtils.h

```diff
@@ -6,16 +6,18 @@
 #define INCLUDED_OCIO_GPUSHADERUTILS_H
 
 #include <sstream>
+#include <utility>
+#include <vector>
```

Comment:
I seems that these includes can be removed. In fact the complete file can be reverted.

Reply:
That's right. One less file in the pull request :+1: